frontend/src/metabase/components/Select.jsx

```diff
@@ -107,6 +112,8 @@ class BrowserSelect extends Component {
           return false;
         } else if (searchCaseInsensitive) {
           return childValue.toLowerCase().startsWith(inputValue.toLowerCase());
+        } else if (searchFuzzy) {
```

Comment:
This doesn't allow you to do both `searchCaseInsensitive` and `searchFuzzy` together.

You could do

```javascript
if (searchCaseInsensitive) {
  childValue = childValue.toLowerCase()
  inputValue = inputValue.toLowerCase()
}
if (searchFuzzy) {
    return childValue.indexOf(inputValue) >= 0;
} else {
  return childValue.startsWith(inputValue);
}
```